frontend/cypress/e2e/side_window_missions.spec.ts

```diff
@@ -37,7 +37,9 @@ context('Missions', () => {
     cy.get('[data-key="DDTM"] > .rs-picker-select-menu-item').click({ force: true })
     cy.get('*[data-cy="Missions-numberOfDisplayedMissions"]').contains('6')
 
-    cy.get('*[data-cy="select-administrations-filter"] > .rs-btn-close').click({ force: true })
+    cy.get('*[data-cy="select-administrations-filter"] > .rs-stack > .rs-stack-item > .rs-btn-close').click({
```

Comment:
All these tests are exactly the kind of cases where you could use monitor-ui `cy.fill()` and it would be (in my opinion) a lot easier to write, to read, and it would remove a lot of data-cy if not all of them. i.e.:

- https://github.com/MTES-MCT/monitorfish/blob/master/frontend/cypress/e2e/side_window/mission_list/filter_bar.spec.ts#L4
- https://github.com/MTES-MCT/monitorfish/blob/master/frontend/cypress/e2e/side_window/mission_form/main_form.spec.ts#L6

`cy.fill()` also accept `undefined` as a value when you want to clear an field.